next/site-config/anvil-portal/footer.ts

```diff
@@ -1,9 +1,70 @@
 import { FooterProps } from "@clevercanary/data-explorer-ui/lib/components/Layout/components/Footer/footer";
+import { SOCIAL } from "@clevercanary/data-explorer-ui/lib/components/common/Socials/socials";
+import logoHhs from "../../images/logoHhs.svg";
+import logoNhgri from "../../images/logoNhgri.svg";
+import logoNih from "../../images/logoNih.svg";
+import logoUsagov from "../../images/logoUsagov.png";
+import { BROWSER_URL } from "./constants";
 
 const footer: FooterProps = {
-  logos: [],
-  navLinks: [],
-  socials: [],
+  logos: [
+    {
+      alt: "nhgri",
+      height: 24,
+      link: "https://www.genome.gov/",
+      src: logoNhgri,
+    },
+    {
+      alt: "nih",
+      height: 24,
+      link: "https://www.nih.gov/",
+      src: logoNih,
+    },
+    {
+      alt: "hhs",
+      height: 32,
+      link: "https://www.hhs.gov/",
+      src: logoHhs,
+    },
+    {
+      alt: "hhs",
+      height: 32,
+      link: "https://www.usa.gov/",
+      src: logoUsagov,
+    },
+  ],
+  navLinks: [
+    {
+      label: "Help",
+      url: `${BROWSER_URL}/help`,
+    },
+    {
+      label: "Privacy",
+      url: `${BROWSER_URL}/privacy`,
+    },
+  ],
+  socials: [
```

Comment:
Now that we have this in twice, let's make a `socials` constant in the `constants.ts` and reference this for both the `header` and `footer` configs.